redis-persistence/src/main/java/com/netflix/conductor/redis/dao/RedisMetadataDAO.java

```diff
@@ -320,6 +321,26 @@ public List<WorkflowDef> getAllWorkflowDefsLatestVersions() {
         return workflows;
     }
 
+    @Override
+    public boolean hasAccess(Object[] args, List<String> labels, String uri) {
+        return false;
+    }
+
+    @Override
+    public boolean exists(Object[] args, String uri) {
+        return false;
+    }
+
+    @Override
+    public List<? extends BaseDef> getUserTaskDefs(List<String> rolesAndGroupsList) {
+        return List.of();
```

Comment:
throw new UnsupportedOperationException("Not implemented yet.");

Reply:
Resolved.